.eslintrc.js

```diff
@@ -0,0 +1,11 @@
+module.exports = {
+    root: true,
+    extends: [
+      '@edwmurph/eslint-config/vue',
+      // uncomment this if you are building a chrome extension
+      // '@edwmurph/eslint-config/webextensions'
+    ],
+    rules: {
+      'vue/no-multiple-template-root': 0
```

Comment:
this rule is bundled into the latest version of my vue eslint config so it shouldn't be needed here if you upgrade `@edwmurph/eslint-config` in your package.json to `3.3.0`